### DISCUSSION TITLE
Question about the example pingpong，help！

### DISCUSSION THREAD

**Discussion by WildStrom** [bwmarrin/discordgo#1247] [Q&A] [answered]
When I am running  the example pingpong, I found the message content received is empty.So whatever I tapped in the discord, no pong respond. Does anybody know the reason?
![image](https://user-images.githubusercontent.com/24494052/190977492-bdbc5e31-5de3-4ef5-979f-2d17b89fe2c3.png)

**Comment by WildStrom** [comment, marked as answer]
I find the solution. there is configuration in the bot management named MESSAGE CONTENT INTENT.

**Reply by VILJkid** [reply]
I was looking to migrate my bot from Py to Go and your question (and solution) were helpful to me. Thanks.
